### DISCUSSION TITLE
Allow user to choose between Pagerduty global and Pagerduty EU endpoint

### DISCUSSION THREAD

**Discussion by guusbeckett** [grafana/grafana#67187] [Alerting]
At our company we use the PagerDuty EU endpoint, but it is not yet possible to use the EU endpoint of PagerDuty. 
As PagerDuty and PagerDuty EU are separate platforms, we cannot simply connect to the global endpoint and expect it to work.

I currently have a custom build that just changes the PagerDuty url to the EU one;
`https://events.pagerduty.com/v2/enqueue` becomes `https://events.eu.pagerduty.com/v2/enqueue`

Is it an idea to provide users with a new alert manager recipient and add _PagerDuty EU_ or would you prefer something else?

I'd be happy to implement this in the code.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82741. Feel free to continue the discussion around this there. Thank you!
